src/components/pagination-nav/pagination-nav.js

```diff
@@ -0,0 +1,200 @@
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import settings from '../../globals/js/settings';
+import mixin from '../../globals/js/misc/mixin';
+import createComponent from '../../globals/js/mixins/create-component';
+import initComponentBySearch from '../../globals/js/mixins/init-component-by-search';
+import handles from '../../globals/js/mixins/handles';
+import on from '../../globals/js/misc/on';
+
+class PaginationNav extends mixin(createComponent, initComponentBySearch, handles) {
+  /**
+   * Pagination Nav component
+   * @extends CreateComponent
+   * @extends InitComponentBySearch
+   * @extends Handles
+   * @param {HTMLElement} element The element working as a pagination nav.
+   */
+  constructor(element, options) {
+    super(element, options);
+    this.manage(on(this.element, 'click', evt => this.handleClick(evt)));
+    this.activePageNumber = this.initActivePageNumber();
+    this.selectorPageSelect = this.element.querySelector(this.options.selectorPageSelect);
+    if (this.selectorPageSelect) {
+      this.manage(on(this.selectorPageSelect, 'change', evt => this.handleSelectChange(evt)));
+    }
+  }
+
+  /**
+   * Get active page number
+   */
+  initActivePageNumber = () => {
+    let pageNum;
+    const activePageElement = this.element.querySelector(this.options.selectorPageActive);
+    if (activePageElement) {
+      pageNum = Number(activePageElement.dataset.page);
+    }
+    return pageNum;
+  };
+
+  /**
+   * Clear active page attributes
+   */
+  clearActivePage = evt => {
+    const selectorPageButtonArray = this.element.querySelectorAll(this.options.selectorPageButton);
+    selectorPageButtonArray.forEach(el => {
+      el.classList.remove(this.options.classActive, this.options.classDisabled);
+      el.disabled = false;
+      el.removeAttribute('data-active');
+      this.anchorAttributesHelper(el);
+    });
+    if (this.selectorPageSelect) {
+      const selectorPageSelectOptions = this.selectorPageSelect.options;
+      for (let i = 0; i < selectorPageSelectOptions.length; i++) {
+        selectorPageSelectOptions[i].removeAttribute('data-active');
+      }
+      if (!evt.target.matches(this.options.selectorPageSelect)) {
+        this.selectorPageSelect.classList.remove(this.options.classActive);
+        this.selectorPageSelect.value = '';
+      }
+    }
+  };
+
+  /**
+   * Add active state on click
+   */
+  handleClick = evt => {
+    this.clearActivePage(evt);
+    const selectorPageElementArray = this.element.querySelectorAll(this.options.selectorPageElement);
+    if (evt.target.matches(this.options.selectorPageButton)) {
+      this.activePageNumber = Number(evt.target.dataset.page);
+    }
+    if (evt.target.matches(this.options.selectorPageDirection)) {
+      if (evt.target.dataset.pageDirection === 'previous') {
+        this.activePageNumber -= 1;
+      }
+      if (evt.target.dataset.pageDirection === 'next') {
```

Comment:
Given we make `[data-page-direction]` selector configurable, let's define `attribPageDirection` option.